index.html

```diff
@@ -52,6 +52,195 @@ <h4 class="social-title">LET'S CONNECT</h4>
           </li>
         </ul>
       </section>
+
+      <div class="works">
+        <div class="card">
+          <header class="card">
+            <img
+              src="./images/snapshot1.png"
+              class="card-header-pic"
+              alt="Projects Header"
+            />
+          </header>
+          <section class="card-projects">
+            <h1 class="card-title">Tonic</h1>
+            <ul class="card-job-desc">
+              <li class="card-company">CANOPY</li>
+              <li><img src="./images/Counter.png" alt="Counter" /></li>
+              <li class="card-job">Back End Dev</li>
+              <li><img src="./images/Counter.png" alt="Counter" /></li>
+              <li class="card-year">2015</li>
+            </ul>
+            <p class="card-persona">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="card-languages">
+              <li class="card-lang1">html</li>
+              <li class="card-lang2">css</li>
+              <li class="card-lang3">javascript</li>
+            </ul>
+            <form action="#" class="card-form">
+              <input
+                type="submit"
+                value="See Project"
+                class="card-submit-button"
+              />
+            </form>
+          </section>
+        </div>
+        <div class="card">
+          <header class="card">
+            <img
+              src="./images/snapshot2.png"
+              class="card-header-pic"
+              alt="Projects Header"
+            />
+          </header>
+          <section class="card-projects">
+            <h1 class="card-title-2">Multi-Post Stories</h1>
+            <ul class="card-job-desc">
+              <li class="card-company">CANOPY</li>
+              <li><img src="./images/Counter.png" alt="Counter" /></li>
+              <li class="card-job">Back End Dev</li>
+              <li><img src="./images/Counter.png" alt="Counter" /></li>
+              <li class="card-year">2015</li>
+            </ul>
+            <p class="card-persona">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="card-languages">
+              <li class="card-lang1">html</li>
+              <li class="card-lang2">css</li>
+              <li class="card-lang3">javascript</li>
+            </ul>
+            <form action="#" class="card-form">
+              <input
+                type="submit"
+                value="See Project"
+                class="card-submit-button"
+              />
+            </form>
```

Comment:
- [ ] It is required to use `<button>` tag for your `See Project` and should be a link with `<a>` tag for semantic reason. `<form>` tag is not the appropriate tag to use at the place.

Reply:
Got it.